doc/nims.rst

```diff
@@ -26,9 +26,25 @@ previous settings):
    ``$project.nim``. This file can be skipped with the same
    ``--skipProjCfg`` command line option.
 
-The VM cannot deal with ``importc`` because the FFI is not
-available. So the stdlib modules using ``importc`` cannot be used with
-Nim's VM. However, at least the following modules are available:
+Limitations
+=================================
+
+NimScript is subject to some limitations caused by the implementation of the VM
+(virtual machine):
+
+* Nim's FFI (foreign function interface) is not available in NimScript. This
+  means that any stdlib module which relies on ``importc`` can not be used in
+  the VM.
+
+* ``ptr`` operations are not available because semantics are hard to emulate
+  with the symbolic representation the VM uses.
+
+* ``var T`` function arguments are not available, as these rely on ``ptr``
```

Comment:
They are available and tested extensively but there are bugs left.

Reply:
Ok